src/status_im/subs/communities.cljs

```diff
@@ -49,6 +50,16 @@
  (fn [[{:keys [members]}] _]
    members))
 
+(re-frame/reg-sub
+ :communities/community-chat-members
+ (fn [[_ community-id]]
+   [(re-frame/subscribe [:communities/community community-id])])
```

Comment:
The sub `:communities/community-members` could be used as an input. The other one for chats I don't know if we have. But it would be a good idea to use these subs as inputs to `:communities/community-chat-members` instead of the whole community.